frontend/src/features/settings/account/ExportAllDataForm.tsx

```diff
@@ -0,0 +1,99 @@
+import React, { useState } from 'react';
+
+import Button from '@material-ui/core/Button';
+import Grid from '@material-ui/core/Grid';
+import LinearProgress from '@material-ui/core/LinearProgress';
+import Typography from '@material-ui/core/Typography';
+import { useSnackbar } from 'notistack';
+
+import { showErrorAlert } from '../../../utils/notifications';
+
+import type { ApiRequestOptions } from 'src/services/openapi/core/ApiRequestOptions';
+import { OpenAPI } from 'src/services/openapi';
+
+// The method below replaces the auto-generated usersMeExportsAllRetrieve
+// from services/openapi/core/request.ts because it does
+// not support returning response.blob which we need for
+// downloading files from the API endpoint /users/me/exports/all
+async function usersMeExportsAllRetrieveBlob(): Promise<Blob> {
+  const api_url = process.env.REACT_APP_API_URL;
+  const options: ApiRequestOptions = {
+    method: 'GET',
+    path: '/users/me/exports/all/',
+  };
+  const access_token =
+    typeof OpenAPI.TOKEN === 'function'
+      ? await OpenAPI.TOKEN(options)
+      : OpenAPI.TOKEN;
+  const headers = { Authorization: `Bearer ${access_token}` };
+  const response = await fetch(`${api_url}${options.path}`, {
+    credentials: 'include',
+    method: options.method,
+    headers: headers,
+  });
+  return response.blob();
+}
```

Comment:
Discussed this with @amatissart and decided to implement our own access to the API endpoint because the autogenerated code does not support `response.blob()`